config/config_test.go

```diff
@@ -65,36 +64,132 @@ environment "prod" "stag" {
 		}
 	}
 }`,
-			seenEnvironments: []string{},
-			seenApplications: []string{},
-			seenSecrets:      []string{},
+		},
+		{
+			name:             "parse service{} with meta",
+			env:              "perf",
+			seenEnvironments: []string{"perf"},
+			seenServices: ConsulServices{
+				{
+					Address: "127.0.0.1",
+					Node:    "test",
+					Service: &api.AgentService{
+						ID:      "test",
+						Service: "test",
+						Tags:    []string{},
+						Port:    1337,
+						Address: "127.0.0.1",
+						Meta: map[string]string{
+							"meta_key": "meta_value",
+						},
+					},
+					Check: &api.AgentCheck{
+						Node:        "test",
+						CheckID:     "service:test",
+						Name:        "test",
+						Status:      "passing",
+						Notes:       "created by hashi-helper",
+						ServiceID:   "test",
+						ServiceName: "test",
+					},
+				},
+			},
+			content: `
+environment "*" {
+	service "test" {
+		address = "127.0.0.1"
+		node    = "test"
+		port    = 1337
+
+		meta {
+			meta_key = "meta_value"
+		}
+	}
+}`,
+		},
+		{
+			name:             "process service{} with 2 meta should fail",
+			env:              "perf",
+			seenEnvironments: []string{"perf"},
+			processErr:       fmt.Errorf("You can only specify meta{} once at -"),
+			content: `
+environment "*" {
+	service "test" {
+		address = "127.0.0.1"
+		node    = "test"
+		port    = 1337
 
-			wantErr: false,
+		meta {
+			meta_key = "meta_value"
+		}
+
+		meta {
+			meta_key_2 = "meta_value_2"
+		}
+	}
+}`,
+		},
+		{
+			name:             "parse service{} with no meta",
```

Comment:
what do we expect with blank meta and multiple meta keys in a single meta

Reply:
Added tests for this